make install failed

Thank you for the great repo! I encountered a problem described below.

## Description

`make install` fails because of missing `FindTBB.cmake`.

```bash
0.505 CMake Error at cmake_install.cmake:95 (file):
0.505   file INSTALL cannot find "/root/small_gicp/cmake/FindTBB.cmake": No such
0.505   file or directory.
```

## Step to Reproduce

Add `RUN make install` after [this line](https://github.com/koide3/small_gicp/blob/ff8269ec09c9ddccff8358a4b7cc5e4c5fda134f/docker/Dockerfile.build.gcc#L19) and build Docker image.

I encountered same problem on my environment ( Ubuntu22.04 (Japanese), VMware17 ).
### Error description
```
my_user_name@computer_name:~/small_gicp/build$ sudo make install
[100%] Built target small_gicp
Install the project...
-- Install configuration: "Release"
-- Up-to-date: /usr/local/include
-- Up-to-date: /usr/local/include/small_gicp
-- Up-to-date: /usr/local/include/small_gicp/benchmark
-- Up-to-date: /usr/local/include/small_gicp/benchmark/benchmark_odom.hpp
-- Up-to-date: /usr/local/include/small_gicp/benchmark/read_points.hpp
-- Up-to-date: /usr/local/include/small_gicp/benchmark/benchmark.hpp
-- Up-to-date: /usr/local/include/small_gicp/util
-- Up-to-date: /usr/local/include/small_gicp/util/normal_estimation.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/vector3i_hash.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/sort_omp.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/downsampling_tbb.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/fast_floor.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/lie.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/normal_estimation_omp.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/downsampling_omp.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/downsampling.hpp
-- Up-to-date: /usr/local/include/small_gicp/util/normal_estimation_tbb.hpp
-- Up-to-date: /usr/local/include/small_gicp/pcl
-- Up-to-date: /usr/local/include/small_gicp/pcl/pcl_point_traits.hpp
-- Up-to-date: /usr/local/include/small_gicp/pcl/pcl_point.hpp
-- Up-to-date: /usr/local/include/small_gicp/pcl/pcl_registration_impl.hpp
-- Up-to-date: /usr/local/include/small_gicp/pcl/pcl_proxy.hpp
-- Up-to-date: /usr/local/include/small_gicp/pcl/pcl_registration.hpp
-- Up-to-date: /usr/local/include/small_gicp/factors
-- Up-to-date: /usr/local/include/small_gicp/factors/icp_factor.hpp
-- Up-to-date: /usr/local/include/small_gicp/factors/plane_icp_factor.hpp
-- Up-to-date: /usr/local/include/small_gicp/factors/general_factor.hpp
-- Up-to-date: /usr/local/include/small_gicp/factors/gicp_factor.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration
-- Up-to-date: /usr/local/include/small_gicp/registration/registration_helper.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration/reduction_tbb.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration/registration.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration/termination_criteria.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration/reduction_omp.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration/rejector.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration/registration_result.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration/optimizer.hpp
-- Up-to-date: /usr/local/include/small_gicp/registration/reduction.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann
-- Up-to-date: /usr/local/include/small_gicp/ann/gaussian_voxelmap.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/kdtree_tbb.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/nanoflann.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/kdtree.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/flat_container.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/nanoflann_tbb.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/traits.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/kdtree_omp.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/nanoflann_omp.hpp
-- Up-to-date: /usr/local/include/small_gicp/ann/incremental_voxelmap.hpp
-- Up-to-date: /usr/local/include/small_gicp/points
-- Up-to-date: /usr/local/include/small_gicp/points/traits.hpp
-- Up-to-date: /usr/local/include/small_gicp/points/point_cloud.hpp
-- Up-to-date: /usr/local/lib/libsmall_gicp.so
-- Up-to-date: /usr/local/lib/cmake/small_gicp/small_gicp-targets.cmake
-- Up-to-date: /usr/local/lib/cmake/small_gicp/small_gicp-targets-release.cmake
-- Up-to-date: /usr/local/lib/cmake/small_gicp/small_gicp-config.cmake
-- Up-to-date: /usr/local/lib/cmake/small_gicp/small_gicp-config-version.cmake
CMake Error at cmake_install.cmake:90 (file):
  file INSTALL cannot find "/home/my_user_name/small_gicp/cmake/FindTBB.cmake": No
  such file or directory.


make: *** [Makefile:86: install] エラー 1
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -274,10 +274,6 @@ if(BUILD_HELPER)
     "${CMAKE_CURRENT_BINARY_DIR}/small_gicp-config-version.cmake"
     DESTINATION ${CMAKE_CONFIG_INSTALL_DIR}
   )
-  install(FILES
-    "${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindTBB.cmake"
-    DESTINATION ${CMAKE_CONFIG_INSTALL_DIR}
-  )
 endif()
 
 if(BUILD_PYTHON_BINDINGS)

